tests/listobjects/listobjects.go

```diff
@@ -103,7 +104,9 @@ func runTests(t *testing.T, schemaVersion string, client ListObjectsClientInterf
 					require.NoError(t, err)
 				}
 
+				// act
```

Comment:
what is act?

Reply:
https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/ :D